microverse-repo1/index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="index.css" rel="stylesheet">
+    <title>Second Repo</title>
+</head>
+<body>
+    <div class="main-body">
```

Comment:
Kindly use html semantics, like header 👍

Reply:
- [ ] Your comment is polite & clear however the reasons for the change is missing. Kindly add a reason why you are requesting this change 👍🏾

Reply:
HTML semantics used for the webpage's Search Engine Optimization (SEO), so it is better if you use semantics, rather than div tags.